src/doc/rustc/src/platform-support/thumbv7em-none-eabi.md

```diff
@@ -50,6 +50,13 @@ to use these flags.
 | Cortex-M7F | SP  | Yes | `cortex-m7` | `-fp64`         |
 | Cortex-M7F | DP  | Yes | `cortex-m7` | None            |
 
+<div class="warning">
+
+Never use the `-fpregs` *target-feature* with these `eabihf` targets
```

Comment:
What does "these" refer to? Should it just be "... with `eabihf` targets ...", or perhaps "... with hard-float ABI targets (such as `eabihf`) ..." ?

Reply:
It should say 'this target'. Broader words can go in the arm target docs.

Reply:
Fixed